masterlist.yaml

```diff
@@ -139,22 +141,26 @@ common:
     util: '[SSEEdit](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
     info: 'A cleaning guide is available [here](https://tes5edit.github.io/docs/5-mod-cleaning-and-error-checking.html#Apendix-B).'
   - &quickClean
-    util: '[SSEEdit](https://github.com/TES5Edit/TES5Edit/releases) **Quick Auto Clean**'
-    info: |+
-
-          >A guide for running **Quick Auto Clean** can be found [here](https://tes5edit.github.io/docs/5-mod-cleaning-and-error-checking.html#quickautoclean).
-          >The latest version of [SSEEdit Experimental](https://github.com/TES5Edit/TES5Edit/releases) is required to use **Quick Auto Clean**.
+    util: '[SSEEdit](https://www.nexusmods.com/skyrimspecialedition/mods/164) **Quick Auto Clean**'
+    info: 'A guide for running **Quick Auto Clean** can be found [here](https://tes5edit.github.io/docs/5-mod-cleaning-and-error-checking.html#quickautoclean).'
   - &reqManualFix
     util: '[SSEEdit](https://www.nexusmods.com/skyrimspecialedition/mods/164) **Quick Auto Clean**'
-    info: 'A cleaning guide is available [here](https://tes5edit.github.io/docs/5-mod-cleaning-and-error-checking.html#Apendix-B). It is strongly recommended not to use mods that contain **Deleted navmeshes** are they''re known to cause crashes. **Deleted navmeshes** must be corrected manually.'
+    info: 'A guide for running **Quick Auto Clean** can be found [here](https://tes5edit.github.io/docs/5-mod-cleaning-and-error-checking.html#quickautoclean). It is strongly recommended not to use mods that contain **Deleted navmeshes** as they''re known to cause crashes. **Deleted navmeshes** must be corrected manually. More information on **Deleted navmeshes** is provided [here](https://tes5edit.github.io/docs/5-mod-cleaning-and-error-checking.html#Apendix-A).'
 
-  - &hasWildEdits
+  - &wildEdits
     type: warn
     content:
       - lang: en
-        text: "This plugin contains wild edits. A manual [SSEEdit](https://www.nexusmods.com/skyrimspecialedition/mods/164) cleaning guide is available here: %1%"
+        text: 'This plugin contains wild edits beyond ITM and UDR records and deleted navmeshes and may require additional manual cleaning to not interfere with other mods.'
+      - lang: da
+        text: 'Dette plugin indeholder ukontrollerede redigeringer udover ITM- og UDR-poster og slettede navmesher og kan behøve yderligere manuel oprydning for ikke at forstyrre andre mods.'
       - lang: de
-        text: "Dieses Plugin enthält unbeabsichtigte Änderungen. Eine manuelle [SSEEdit](https://www.nexusmods.com/skyrimspecialedition/mods/164)-Säuberungsanleitung ist hier verfügbar: %1%"
+        text: 'Dieses Plugin enthält wilde Modifikationen weit über ITM-, und UDR Einträge und entfernte Navmeshes hinaus und benötigt vermutlich mehr manuelle Säuberung um andere Mods nicht zu beeinträchtigen.'
+  - &cleaningGuide
```

Comment:
Same as below
>I could make a post in AFK forums for wild edit cleaning guides, which would be separate to the >LOOT page. I'm not sure where else would be appropriate to host them.

Reply:
@Ortham I going to go with appending the guides as a sub.
![image](https://user-images.githubusercontent.com/1944639/48585165-aff50e00-e923-11e8-9dfb-efe9317d732e.png)

I've tried adding it to the cleaning info messages, but you'll still have the `xEdit found dirty edits` message, which seems out of place since it's manual cleaning.

Reply:
Updated in https://github.com/loot/skyrimse/pull/468/commits/bbf821e16285ff6a1764b402bea99665fedb6fa4